core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -560,6 +563,113 @@ export namespace RealityModelTileTree {
 
       if (undefined !== props.classifiers)
         this._classifier = createClassifierTileTreeReference(props.classifiers, this, props.iModel, props.source);
+
+      if (undefined !== props.realityDataDisplayStyle) {
+        this._clips = props.realityDataDisplayStyle.clips;
+      }
+
+      if (undefined !== props.modelId) {
```

Comment:
Check if model Id is valid Id and is not a transient Id - see `Id64.isValidId64` and `Id64.isTransient`.

Reply:
Done! See 0378dfb